docs/predictive-analytics/next-best-offer/manual.sql

```diff
@@ -0,0 +1,24 @@
+-- Manually insert data if source connectors are not run
+INSERT INTO CUSTOMERS (customer_id, first_name, last_name, email, gender, income, fico) VALUES  (1,'Waylen','Tubble','wtubble0@hc360.com','Male',403646, 465);
```

Comment:
For consistency with stylization of other recipes, these stream/table names should all be lowercase.

CUSTOMERS --> customers
CUSTOMER_ACTIVITY_STREAM --> customer_activity_stream

etc